pulp_file/tests/functional/api/test_crud_content_unit.py

```diff
@@ -98,6 +105,70 @@ def test_04_delete(self):
         self.assertEqual(exc.exception.response.status_code, 405)
 
 
+class ContentUnitUploadTestCase(unittest.TestCase):
+    """CRUD content unit with upload feature."""
```

Comment:
It would be nice to add the related issue here. 
```suggestion
    """CRUD content unit with upload feature.
    
    This test targets the following issue:
    
    `Pulp #5403 <https://pulp.plan.io/issues/5403>`_
    """
```